### PR TITLE
Updated Syntax

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
#### Summary

Replaced var with const or let as appropriate & updated spreadsheet as some files contain Var missing in the list.
#### Related issues

Resolving #16662
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by PrajwalBorkar** [comment]
Does anyone know how to resolve this error? I'm getting this when I try to make changes in repo.
Screenshot (69)

File: files/en-us/web/api/mssitemodeevent/index.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Although this event inherits from the [Event](/en-US/docs/Web/API/Event) object,
 ## Example
 
 ```
-declare var MSSiteModeEvent: {
+declare const MSSiteModeEvent: {
```
Comment (teoli2003) [inline files/en-us/web/api/mssitemodeevent/index.md:156]: This is not JavaScript.

```suggestion
declare var MSSiteModeEvent: {
```

File: files/en-us/web/api/navigator/productsub/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ A string.
 ```js
 <script>
```
Comment (teoli2003) [inline files/en-us/web/api/navigator/productsub/index.md:26]: Let's transform this into JS:
```suggestion
```

File: files/en-us/web/api/navigator/productsub/index.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ A string.
...
 function prodsub() {
-  var dt = document.getElementById("d").childNodes[0];
+  const dt = document.getElementById("d").childNodes[0];
   dt.data = navigator.productSub;
 }
 </script>
```
Comment (teoli2003) [inline files/en-us/web/api/navigator/productsub/index.md:31]:
```suggestion
console.log(prodsub()); // Returns: 20010725
```

Please delete lines 32-34

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One example is not real JS, let's fix it.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**